.github/workflows/codeql-analysis.yml

```diff
@@ -37,6 +37,11 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v2
 
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.18
```

Comment:
Same change was made in objstore. Required for codeql to pass with libraries using generics.